packages/frontend/src/screens/app/screens/tournaments/screens/tournament/screens/match/match-data.tsx

```diff
@@ -25,12 +25,25 @@ const MATCH_INFO_QUERY = gql`
 				name
 				players
 			}
+			tournament {
+				id
+				owner {
+					id
+				}
+			}
 			contested
+			finalized
 		}
 	}
 `;
 
-const MatchData = () => {
+interface Props {
+	tournament: {
+		id: string;
+	};
+}
+
+const MatchData: React.FC<Props> = () => {
 	const { matchId } = useParams();
```

Comment:
is useParams() a "magic" function in GraphQL? Or one we have defined?

Reply:
I am not completly familiar with use params but my quick google search might clear it up :P 
useParams is not a GQL function but a React one from the React-Router-Dom package(the package handeling which screens to show and how to link our app together). From what I can tell useParams returns an object with all URL params, like /tournaments/***SOME UUID***. Here we destructure that object and take out the matchId from the current url. An example would be localhost:3000/tournaments/1234(tournamentid)/match/4567(matchId) where { matchId } would give us 4567. I am NOT 100 sure though

Reply:
It's worth noting that useParam only returns the params in the current switch, so params in different nested routes cannot be accessed